pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -90,7 +99,7 @@ public static FileUploadType getDefaultUploadType() {
   private static final String HTTP = "http";
   private static final String HTTPS = "https";
   private static final String SCHEMA_PATH = "/schemas";
-  private static final String SEGMENT_PATH = "/segments";
+  private static final String SEGMENT_PATH = "/v2/segments";
```

Comment:
Has the old path been removed? The idea is to have both in place, right? Maybe I am missing something here. The idea is not to introduce a new header, but a new protocol endpoint.

Reply:
thanks,